2020-02-Spring/1_strings/README.md

```diff
@@ -13,11 +13,11 @@ In the style of:
 * [Problems](#problems)
   * [1](#p1)
   * [2](#p2)
-  * [3](#p3)
+  * [3](#p3) Add Strings
```

Comment:
I've never done this, but I think it would be fine. Just make sure that the problem names are in place for all of the problems & solutions, not just your own. (Coordinating with other contributors on this point is recommended.)